apps/theming/lib/ImageManager.php

```
@@ -165,6 +187,12 @@ public function delete(string $key) {
 		} catch (NotFoundException $e) {
 		} catch (NotPermittedException $e) {
 		}
+		try {
+			$file = $this->appData->getFolder('images')->getFile($key . '.png');
+			$file->delete();
+		} catch (NotFoundException $e) {
+		} catch (NotPermittedException $e) {
+		}
```

I'd prefer having a comment stating why we ignore the exception here ;)

---

Seconded